### DISCUSSION TITLE
borgmatic

### DISCUSSION THREAD

**Discussion by sn0opy** [selfhst/icons#3] [New Icon Request]
https://torsion.org/borgmatic/

Logo in SVG here: https://torsion.org/borgmatic/docs/static/borgmatic.svg
Or here: https://projects.torsion.org/borgmatic-collective/borgmatic/src/branch/main/docs/static/borgmatic.svg

**Comment by shollyethan** [comment]
borgmatic has been added. Thanks for sharing the links - it definitely makes my life easier.
